Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -12278,6 +12308,12 @@ void WebPageProxy::generateTestReport(const String& message, const String& group
     send(Messages::WebPage::GenerateTestReport(message, group));
 }
 
+void WebPageProxy::addSubframePageProxyForFrameID(WebCore::FrameIdentifier frameID, WebCore::RegistrableDomain domain, std::unique_ptr<SubframePageProxy>&& subframePageProxy)
+{
+    m_subframePageProxyForDomain.add(domain, WTFMove(subframePageProxy));
```

Comment:
We will eventually need a corresponding remove call.  Doesn't need to be in this PR, but worth adding a FIXME comment for both add calls.